tests/js-custom-authorization/custom_authorization.py

```diff
@@ -569,15 +569,22 @@ def test_datetime_api(network, args):
         local_time = datetime.datetime.now(datetime.timezone.utc)
         assert r.status_code == http.HTTPStatus.OK, r
         body = r.body.json()
-        assert body["default"] == body["definitely_now"], body
+
         # Python datetime "ISO" doesn't parse Z suffix, so replace it
+        default = body["default"].replace("Z", "+00:00")
         definitely_now = body["definitely_now"].replace("Z", "+00:00")
+        definitely_1970 = body["definitely_1970"].replace("Z", "+00:00")
```

Comment:
Technically of course, it could be "z": https://www.rfc-editor.org/rfc/rfc3339#section-5.6, although not from an ECMAScript compliant implementation: https://tc39.es/ecma262/multipage/numbers-and-dates.html#sec-date-time-string-format